CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -1045,8 +1047,8 @@ protected function processFormContribution(
 
       $contribution = CRM_Contribute_BAO_Contribution::add($contributionParams);
 
-      // lets store it in the form variable so postProcess hook can get to this and use it
-      $form->_contributionID = $contribution->id;
+      // lets store it in the form variable so postProcess hook can access it via getContributionID()
+      $this->_contributionID = $contribution->id;
```

Comment:
`$this` & `$form` are the same in this function - need to switch them all to `$this` in another clean up pass